### ISSUE TITLE
AWS: tags are generated in wrong way for security groups

### ISSUE THREAD

**Issue by sirmax123** [GoogleCloudPlatform/terraformer#119]
Software versions:
```
$ terraform version
Terraform v0.12.2
+ provider.aws v2.16.0
```
```
$ terraformer version
Terraformer v0.7.5
```
(installed with `brew`)

Steps to reproduce:

```
terraformer \
    import  \
    aws \
    --resources=sg \
    --regions=some-region
```

Actual result:
```
  tags  {
    Name = "<some name here>"
  }

```
Expected result according to the **terraform**  documentation (https://www.terraform.io/docs/providers/aws/r/security_group.html)
```
  tags = {
    Name = "<some nema here>"
  }
```

Workaround:
replace with sed or other tool `tags {` --> `tags = { `

**Comment by sergeylanzman** [comment]
it's terraform 0.12 issue. 
I tested terraform 0.12 in beta version with old provide(before 2.7) but now it's not working. 
Need more changes for 0.12 
https://github.com/GoogleCloudPlatform/terraformer/commit/43cef73abf2698ea8e10e4aa3139b7b7d4a1cab9

**Comment by fbongiovanni29** [comment]
@sergeylanzman Are there any plans on adding a fix for this? If so, is there anything I could help with? If you point me in the right direction I might be able to open a PR

**Comment by meshuga** [comment]
@fbongiovanni29 
It's not an easy thing as just making a quick hack, the issue is related with move to Terraform 0.12, which introduces a lot of internal Terraform changes as well as hacks. I still try to work on https://github.com/meshuga/terraformer/tree/tf_12 to provide move to TF 0.12 but had to remove several providers in order to make the solution working. I managed to provide workable solution for AWS (at least hope so) although all other providers must be rechecked to verify they are working as expected + some of them must be reintroduced.

**Comment by fbongiovanni29** [comment]
Thanks for the update!

**Comment by sergeylanzman** [comment]
Merged to master support 0.12 include support aws tags
